qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -0,0 +1,306 @@
+import logging
+import time
+import re
+
+from virttest import error_context
+from virttest import utils_misc
+from virttest import data_dir
+from virttest.utils_windows import virtio_win, wmic
+
+
+QUERY_TIMEOUT = 360
+INSTALL_TIMEOUT = 360
+OPERATION_TIMEOUT = 120
+
+
+def _pnpdrv_info(session, name_pattern, props=None):
+    cmd = wmic.make_query("path win32_pnpsigneddriver",
+                          "DeviceName like '%s'" % name_pattern,
+                          props=props, get_swch=wmic.FMT_TYPE_LIST)
+    return wmic.parse_list(session.cmd(cmd, timeout=QUERY_TIMEOUT))
+
```

Comment:
same function with https://github.com/autotest/tp-qemu/pull/2482
We should reuse code